Gemfile

```diff
@@ -76,6 +77,8 @@ group :development, :test do
   gem 'rspec-rails'
   # Factory bot: factories for testing
   gem 'factory_bot_rails'
+  # Shoulda matchers
```

Comment:
Can probably have a better comment here. In particular, say that it's an experimental gem running Rails 5 (basically explain why you need the `rails-5` branch).